Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -137,7 +137,8 @@ bool AVIDump::CreateVideoFile()
     return false;
   }
 
-  const AVCodecDescriptor* codec_desc = avcodec_descriptor_get_by_name(g_Config.sDumpCodec.c_str());
+  const std::string& codec_name = g_Config.bUseFFV1 ? "ffv1" : g_Config.sDumpCodec;
```

Comment:
Instead of keeping the `UseFFV1` config variable around, I think it makes more sense to change the GUI to set `DumpCodec = "ffv1"` when that checkbox is clicked.